app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -182,6 +170,15 @@ private void openContact(final int index) {
     }
 
     /**
+     * Starts the QR Exchange Activity.
+     */
+    private void startQRExchangeActivity() {
```

Comment:
Doesn't this do the same as [onQRButtonClicked](https://github.com/ericcornelissen/NervousFish/pull/169/files#diff-149538cc1f1f5e4370b6ae93d6489330R129)